### PR TITLE
Add JetStream basic usage to readme

### PR DESCRIPTION
**Pull request by variadico**
Add JetStream basic usage to readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -423,6 +422,65 @@ resp := &response{}
...
 
+## JetStream Basic Usage
+
+```go
+import nats "github.com/nats-io/nats.go"
+
+// Connect to NATS
+nc, _ := nats.Connect(nats.DefaultURL)
+
+// Connect to JetStream
```
Comment (derekcollison) [inline README.md:433]: Would say `create jetstream context`

File: README.md [hunk 2]
```diff
@@ -423,6 +422,65 @@ resp := &response{}
...
+js, _ := nc.JetStream()
+
```
Comment (derekcollison) [inline README.md:435]: Show both management and normal as separate sections?

File: README.md [hunk 3]
```diff
@@ -423,6 +422,65 @@ resp := &response{}
...
+js.Subscribe("ORDERS.*", func(m *nats.Msg) {
+	fmt.Printf("Received a JetStream message: %s\n", string(m.Data))
+})
+
+// Simple Sync Durable Consumer (optional SubOpts at the end)
+sub, err := js.SubscribeSync("ORDERS.*", nats.Durable("MONITOR"), nats.MaxDeliver(3))
+m, err := sub.NextMsg(timeout)
+
+// Simple Pull Consumer
+sub, err := js.PullSubscribe("ORDERS", "MONITOR")
```
Comment (derekcollison) [inline README.md:468]: First arg the stream name vs subject?

**Review by derekcollison — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (variadico) [reply README.md]: Ah, yep. Typo here. Meant `ORDERS.*`.

File: README.md (hunk 2 above)
Comment (variadico) [reply README.md:435]: Split into 2 sections: JetStream Basic Usage and JetStream Basic Management.

File: README.md [hunk 4]
```diff
@@ -423,6 +422,82 @@ resp := &response{}
 err := c.RequestWithContext(ctx, "foo", req, resp)
 ```
 
+## JetStream Basic Usage
+
+```go
+import nats "github.com/nats-io/nats.go"
```
Comment (derekcollison) [inline README.md:428]: Don't need nats here, Go does right thing. Your editor put that in but its not needed.

File: README.md [hunk 5]
```diff
@@ -423,6 +422,82 @@ resp := &response{}
...
+
+// Connect to NATS
+nc, _ := nats.Connect(nats.DefaultURL)
```
Comment (derekcollison) [inline README.md:431]: If its default URL is it needed? I can't remember..

File: README.md [hunk 6]
```diff
@@ -423,6 +422,82 @@ resp := &response{}
...
+
+// Create JetStream Context
+js, _ := nc.JetStream()
+
+// Simple Stream Publisher
+js.Publish("ORDERS.scratch", []byte("hello"))
+
+// Simple Async Stream Publisher
+paf, err := js.PublishAsync("ORDERS.scratch", []byte("hello"))
```
Comment (derekcollison) [inline README.md:440]: I might do the batch example which is what people would copy for a benchmark.

File: README.md [hunk 7]
```diff
@@ -423,6 +422,82 @@ resp := &response{}
...
+
+// Create JetStream Context
+js, _ := nc.JetStream()
+
+// Create a Stream
+js.AddStream(&nats.StreamConfig{
+	Name:     "ORDERS",
+	Subjects: []string{"ORDERS.*"},
+})
+
```
Comment (derekcollison) [inline README.md:488]: Show UpdateStream IMO as well.

**Review by derekcollison — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (variadico) [reply README.md:428]: Ah, ok. I'll update the existing example too then. 👍🏽

File: README.md (hunk 5 above)
Comment (variadico) [reply README.md:431]: The function requires a string parameter, but we can pass an empty string for the default. Will update existing samples too.

Comment (derekcollison) [reply README.md:431]: No need, that is good.

**Comment by variadico** [comment]
All right. Added UpdateStream and async batch publish.

File: README.md [hunk 8]
```diff
@@ -33,10 +33,10 @@ go get github.com/nats-io/nats-server/v2
 ## Basic Usage
 
 ```go
-import nats "github.com/nats-io/nats.go"
+import "github.com/nats-io/nats.go"
 
 // Connect to a server
-nc, _ := nats.Connect(nats.DefaultURL)
+nc, _ := nats.Connect("")
```
Comment (derekcollison) [inline README.md:39]: Would put back to the way before.

**Review by derekcollison — APPROVED** [review]
LGTM - One minor comment.

File: README.md [hunk 9]
```diff
@@ -423,6 +422,85 @@ resp := &response{}
...
+sub.Drain()
+```
+
+## JetStream Basic Management
+
+```go
+import "github.com/nats-io/nats.go"
+
+// Connect to NATS
+nc, _ := nats.Connect("")
```
Comment (wallyqs) [inline README.md:475]: Think here just change back to:
```suggestion
nc, _ := nats.Connect(nats.DefaultURL)
```

**Pull request merged**